apps/auth/oidc/__init__.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013 - 2016 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import logging
+
+import superdesk
+
+from .auth import OIDCAuthResource, OIDCAuthService
+
+logger = logging.getLogger(__name__)
+
+
+def init_app(app):
+    endpoint_name = 'auth_oidc'
+    if bool(app.config['OIDC_ENABLED']) and not bool(app.config['SECRET_KEY'] != ''):
```

Comment:
why do we need `bool` here?
`app.config['OIDC_ENABLED']` is bool anyway and `!=` returns a bool